src/app/shared/http-services/get-group-descendants.service.ts

```diff
@@ -57,7 +57,7 @@ export class GetGroupDescendantsService {
 
```

Comment:
When switching between "direct children" and "all descendants", requests are triggered twice, can you have a look?
![image](https://user-images.githubusercontent.com/16257159/178440744-8b922950-d65c-4942-8a04-4ceaabcf4108.png)

If it's straightforward to fix, can you fix it, and if not, create an issue?

Reply:
Already have [issue](https://github.com/France-ioi/AlgoreaFrontend/pull/1098) for this